cli/src/semgrep/commands/ci.py

```diff
@@ -441,6 +442,7 @@ def ci(
             optimizations=optimizations,
             baseline_commit=metadata.merge_base_ref,
             baseline_commit_is_mergebase=True,
+            dump_contributions=True,
```

Comment:
so this is done for all CI jobs ... note that git log can take some time on huge repo. Are people concerned about the privacy of this? Did we add stuff in PRIVACY.md about the fact we inspect all the history of a repo and send all this information? for example we send the email of all those contributors ...
cc @chmccreery @milanwilliams

Reply:
According to https://semgrep.dev/docs/usage-limits/ we are interested only in the last 30 days of a repository, so maybe we should limit the git log we send about commits in the last 30 days @zyannes ?
In theory we don't really care about their actual name, just the count, so we could even locally compute the count so for privacy we would just need to send a number (and maybe the hash of the commit_name and email)?
What do you think @zyannes @milanwilliams @chmccreery ?

Reply:
Yes, @ThatDevShparki added this to the PRIVACY.md about two weeks ago: https://github.com/returntocorp/semgrep/commit/6976ee24bbc7eebc22a5d21a6e3b16e33fa45910.

Reply:
@aryx yes, good call on the 30-day limit. 

I believe we do need their actual name and email so the contributor count can be accurately calculated. This information will also help resolve any contributor count disputes.

Reply:
@aryx Just opened [this PR](https://github.com/returntocorp/semgrep/pull/8713) to limit to 30 days.